resources/views/components/frameworks/tailwind/filters/date-picker.blade.php

```diff
@@ -6,7 +6,6 @@
     'tableName' => null,
 ])
 @php
-    $tableName = \Illuminate\Support\Str::kebab($tableName);
```

Comment:
Conversion to kebab case affects listeners in [PowerGridComponent](https://github.com/Power-Components/livewire-powergrid/blob/main/src/PowerGridComponent.php#L516). Another option would be to convert table name to kebab case in `powerGridListeners` method.